server/src/main/java/org/eclipse/openvsx/entities/UserData.java

```diff
@@ -63,6 +63,7 @@ public UserJson toUserJson() {
         json.avatarUrl = this.getAvatarUrl();
         json.homepage = this.getProviderUrl();
         json.provider = this.getProvider();
+        json.role = this.getRole();
```

Comment:
This method is used in multiple places, e.g. displaying the user who published an extension, or the user who wrote a review. The role should not be exposed in those cases! Please move this line to the `UserAPI.getUserData` endpoint, which returns the data of the currently logged in user.